org.kde.kanagram.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "org.kde.kanagram",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "finish-args": ["--share=network", "--socket=pulseaudio", "--share=ipc", "--socket=x11", "--socket=wayland" ],
```

Comment:
I think we need       "--device=dri",
to get away from that ugly warning when starting

libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: El fitxer o directori no existeix
libGL error: failed to load driver: i965